### ISSUE TITLE
Simplify issue template

### ISSUE THREAD

**Issue by ratanparai** [techcombd/academia#44]
## I'm submitting a...

<pre><code>
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[x] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

## Current behavior

Current issue template seems complex for some. Those who are new to github is having trouble following instruction. 

## Expected behavior

Make the issue template simple so that everyone can create issue without going through a lots of trouble

## What is the motivation / use case for changing the behavior?

To get more interaction from new developers in this repository

**Comment by ratanparai** [comment]
My personal opinion is keeping the `issue template` as it is now. Reading and following instruction is not that hard. If someone don't want to give time for reading/writing issues, he/she would be a burden in the project. 

I am not comfortable wasting times for people who don't have time to read in the first place

**Comment by mahedee** [comment]
Yes, I think, its fine. I am not clear about "What is the motivation / use case for changing the behavior?" section? Is it possible to keep this title short, simple and meaning full. What if add another optional section name "Description/Issue Description/Details"? Suppose I want to say, model should be like this, page should be like this etc.

**Comment by ratanparai** [comment]
@mahedee: `What is the motivation/ use case for changing the behavior?` section has literal meaning. 

From dictionary -
> `Motivation`: a reason or reasons for acting or behaving in a particular way
> `use case` : a use case is a list of actions or event steps typically defining the interactions between a role  and a system to achieve a goal 

So it means, What is the reason or why you want to change the current behavior `AND` how this change can be useful?

>PS. I am not good at describing meaning of words

I am trying to give you an example related to your sample question-


## I'm submitting a...

<code>

[ ] Bug report

[ ] Performance issue

[x] Feature request

[ ] Documentation issue or request

[ ] Other... Please describe:

</code>

## Current behavior
`Institute` entity do not have `added date`  property

## Expected behavior
There should be a `added date` property in `Institute` entity. This will be inserted automatically when the institute is created and will be unchanged in the institute entity's whole lifecycle. 

## What is the motivation / use case for changing the behavior?
When generating report we need to show when the institute was added. Moreover we can use the date for giving them gifts on like `5 years with us` etc
